cdap-ui/app/cdap/components/Card/index.js

```diff
@@ -131,5 +131,6 @@ Card.propTypes = {
   size: PropTypes.oneOf(['SM', 'MD', 'LG']),
   cardStyle: PropTypes.object,
   onClick: PropTypes.func,
-  id: PropTypes.string
+  id: PropTypes.string,
+  showTitle: PropTypes.bool
```

Comment:
Do we need a boolean for this? Isn't the card already checking for a title and if its present then show? Are we having a scenario where we have to show and hide title in the card? Seems like something is missing here.